appservice/src/connectToGitHub.ts

```diff
@@ -22,7 +22,7 @@ type gitHubLink = { prev?: string, next?: string, last?: string, first?: string
 // tslint:disable-next-line:no-reserved-keywords
 type gitHubWebResource = WebResource & { resolveWithFullResponse?: boolean, nextLink?: string, lastLink?: string, type?: string };
 
-export async function connectToGitHub(node: IAzureNode, client: SiteClient): Promise<void> {
+export async function connectToGitHub(node: AzureTreeItem, client: SiteClient): Promise<void> {
```

Comment:
nit: parameter is still called `node`.  Few other cases in this PR

Reply:
I left everything called `node` everywhere except the UI package. Didn't seem worth all the work/review